src/cpp/session/modules/SessionSVN.cpp

```diff
@@ -1190,7 +1190,7 @@ struct CommitInfo
    std::string author;
    std::string subject;
    std::string description;
-   boost::posix_time::time_duration::sec_type date;
+   boost::uint64_t date;
```

Comment:
Why was this change necessary? (what compile error were you seeing with Boost 1.65?)

FWIW it looks like `sec_type` changed from `int32_t` in Boost 1.63.0 to `std::time_t` in Boost 1.65.0.

Reply:
I got an ambiguous convention error from `sec_type` to `value_type` in [this line](https://github.com/randy3k/rstudio/blob/90701c01101e96933729f57f2f39364d60a31162/src/cpp/session/modules/SessionSVN.cpp#L1436), likely that the json array `pDates ` [doens't understand `sec_type`].